src/app/views/query-runner/util/iframe-message-parser.ts

```diff
@@ -0,0 +1,111 @@
+function isVerb (word: string): boolean {
+  return ['GET', 'POST', 'PUT', 'PATCH', 'DELETE'].indexOf(word) !== -1;
+}
+
+function isUrl(word: string): boolean {
+  return word.includes('https') && !word.includes('{');
+}
+
+function isHeaderKey(word: string): boolean {
```

Comment:
Will we only ever have the Content-type header?

Reply:
That's what I have seen in the documentation snippets. Are there any other headers?

Reply:
Yes. For example, in todays meeting, we saw the Prefer header being used.

Reply:
You stated that you have this fix but this is on hold until we get classy GE all gussied up.